### PR TITLE
Vschema and Query Serving Review

### LINKED ISSUE #1232: [Docs Review] VSchema and Query Serving
Parent Issue vitessio/website#1222 
Vitess github url: https://github.com/vitessio/website

### Pages to review:

- [x] [VSchema and Query Serving](https://vitess.io/docs/15.0/user-guides/vschema-guide/)
- [x] [Overview](https://vitess.io/docs/16.0/user-guides/vschema-guide/overview/)
- [x] [Sharding Guidelines](https://vitess.io/docs/16.0/user-guides/vschema-guide/sharding-guidelines/)
- [x] [Unsharded Keyspace](https://vitess.io/docs/16.0/user-guides/vschema-guide/unsharded/)
- [x] [Sharded Keyspace](https://vitess.io/docs/16.0/user-guides/vschema-guide/sharded/)
- [x] [Sequences](https://vitess.io/docs/16.0/user-guides/vschema-guide/sequences/)
- [x] [Shared Vindexes and Foreign Keys](https://vitess.io/docs/16.0/user-guides/vschema-guide/shared-vindexes/)
- [x] [Unique Lookup Vindexes](https://vitess.io/docs/16.0/user-guides/vschema-guide/unique-lookup/)
- [x] [Non-Unique Lookup Vindexes](https://vitess.io/docs/16.0/user-guides/vschema-guide/non-unique-lookup/)
- [x] [Lookup as Primary Vindex](https://vitess.io/docs/16.0/user-guides/vschema-guide/lookup-as-primary/)
- [x] [Backfill Lookup Vindexes](https://vitess.io/docs/16.0/user-guides/vschema-guide/backfill-vindexes/)
- [x] [Subsharding Vindex](https://vitess.io/docs/16.0/user-guides/vschema-guide/subsharding-vindex/)
- [x] [Advanced VSchema Properties](https://vitess.io/docs/16.0/user-guides/vschema-guide/advanced-vschema/)
- [x] [Pictorial Summary](https://vitess.io/docs/16.0/user-guides/vschema-guide/pictorial/)

### What to do 

- [x] Make sure the content reflect the current state and facts are up-to-date.
- [x] Add details if needed either in form of new section on current page or adding new page.
- [x] Add links/Urls on page are working
- [x] Deprecate flags should be marked as 'deprecated'
- [x] Review is done for both V15 and V16

Please note that we are not going to focus of restructuring the website content as part of this project. It is something we will do as a separate project later on.

### PR DESCRIPTION
**Pull request by GuptaManan100**
### Description

This PR reviews the Vschema and query serving section of the website

### Related Issues
- Fixes #1232

### CODE REVIEW

File: content/en/docs/16.0/user-guides/vschema-guide/sharding-guidelines.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ If your application is tenant-based, it is possible that a single tenant may gro
 
 The question to ask oneself is: if the tenant were a single application by themselves, what would be their sharding key, and then shard by that key instead of the tenant id.
 
-Vitess has started rolling out support for multi-column Vindexes. Once this feature is fully done, you will be able to shard by the tenant id and a secondary key. The two-column sharding approach will allow you to group all data for a given tenant into a smaller set of shards rather than a random distribution. This may be beneficial for security or compliance reasons, in case the tenant would want their data to be physically isolated from other tenants.
+Vitess now has support for multi-column Vindexes. You can now shard by the tenant id and a secondary key. The two-column sharding approach allows you to group all data for a given tenant into a smaller set of shards rather than a random distribution. This may be beneficial for security or compliance reasons, in case the tenant would want their data to be physically isolated from other tenants.
```
Comment (GuptaManan100) [inline content/en/docs/16.0/user-guides/vschema-guide/sharding-guidelines.md]: We don't have any docs for multi-column vindex. We can link them here when we add them.

File: content/en/docs/15.0/user-guides/vschema-guide/overview.md [hunk 2]
```diff
@@ -29,6 +29,6 @@ You can also use the demo app to follow the steps of this user guide. If so, you
```
Comment (rsajwani) [inline content/en/docs/15.0/user-guides/vschema-guide/overview.md:29]: and that server gives you the semblance of being
>>> perhaps its resemblance ?

File: content/en/docs/16.0/user-guides/vschema-guide/sharding-guidelines.md (hunk 1 above)
Comment (GuptaManan100) [reply content/en/docs/16.0/user-guides/vschema-guide/sharding-guidelines.md]: I was wrong, we have them. I'll add a link.

**Review by rsajwani — APPROVED** [review]

File: content/en/docs/15.0/user-guides/vschema-guide/overview.md (hunk 2 above)
Comment (GuptaManan100) [reply content/en/docs/15.0/user-guides/vschema-guide/overview.md:29]: No, I think semblance is fine. I ran it through grammar check just to be sure.

**Pull request merged**